presto-product-tests/conf/docker/singlenode-simple-no-impersonation/docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
     command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode run
     volumes:
       - ../../../../:/docker/volumes/presto
+      - ../../../../presto-product-tests/conf/tempto/tempto-configuration.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
```

Comment:
Maybe we should rename the `tempto-configuration.yaml` into `tempto-configuration-for-docker-default.yaml`. And then rename the `tempto-configuration-kerberized.yaml` into `tempto-configuration-docker-kerberos.yaml`. So it is more intuitive which configuration is used in the simplest case. What do you think?